python/repositories.bzl

```diff
@@ -187,10 +185,18 @@ python_repository = repository_rule(
             mandatory = True,
             values = TOOL_VERSIONS.keys() + MINOR_MAPPING.keys(),
         ),
+        "release_filename": attr.string(
```

Comment:
hmm it's smelly there's no docs/ change in this PR, something must not be wired up